code/go/0chain.net/smartcontract/dbs/event/error.go

```diff
@@ -0,0 +1,10 @@
+package event
+
+type Error struct {
+	TransactionID string
+	Error         string
+}
+
+func (edb *EventDb) addError(err Error) error {
```

Comment:
@Sriep @kushthedude Don’t we need logic to remove duplicates here?
Or overwrite records ?

Reply:
We want to avoid duplicates, however, error rows should be immutable so no overwriting.